lib/spack/spack/cmd/install.py

```diff
@@ -90,10 +90,15 @@ def setup_parser(subparser):
         help="spec of the package to install"
     )
     subparser.add_argument(
-        '--run-tests', action='store_true', dest='run_tests',
+        '--test-all', action='store_true', dest='test_all',
```

Comment:
What do you think about having  a single argument `--test` that could take values `all` and `root`? This will make the 2 options mutually exclusive, and allow for seamless extensions in the future without adding additional arguments.

Reply:
I like the idea.

Reply:
I agree - I'll try to add this today